### PR TITLE
Attach fn-name metadata to instrumented CLJS functions

### PR DESCRIPTION
**Pull request by ingesolvoll**
Our app uses function values for looking up their original name/symbol from a registry. It is not something we can or want to change, and using malli instrumentation for CLJS breaks this. The reason is that after instrumentation, the anonymous wrapper has a different identity than the original function.

If malli could attach the original symbol as metadata, we are able to use this as an alternative way of obtaining the same thing.

I used the key `:fn-name` as that was used elsewhere in the same code. Possible alternatives:
- `:instrumented-symbol`
- `:instrumented-var`
- `:instrumented-fn`

### CODE REVIEW

File: src/malli/instrument/cljs.cljs [hunk 1]
```diff
@@ -7,5 +7,14 @@
 (defn -filter-var [f] (fn [_ s _] (f s)))
 (defn -filter-ns [& ns] (fn [n _ _] ((set ns) n)))
 
+(defn meta-fn
```
Comment (ingesolvoll) [inline src/malli/instrument/cljs.cljs:10]: Stolen from https://github.com/CrypticButter/snoop/blob/main/src/crypticbutter/snoop/impl/cljs.cljs

**Comment by ikitommi** [comment]
As it's a symbol, I think `:fn-name` is misleading. I like the `:instrumented-symbol` (it's not a Var either, or not a name).

**Comment by ingesolvoll** [comment]
@ikitommi Thanks, I updated the name.

**Comment by ikitommi** [comment]
Awesome, thanks!

**Pull request merged**